library/Faktory/Worker.hs

```diff
@@ -5,16 +5,18 @@
 --
 module Faktory.Worker (
   WorkerHalt (..),
-  WorkerConfig (..),
+  Worker,
+  workerID,
```

Comment:
Rather than rewriting selectors to expose, you can just selectively expose the selectors from the type itself:

```suggestion
  Worker(workerID),
```